ui/media/js/main.js

```diff
@@ -712,6 +718,8 @@ async function checkTasks() {
 
     task.isProcessing = false
     task['stopTask'].innerHTML = '<i class="fa-solid fa-trash-can"></i> Remove'
+    task['useSettings'].innerHTML = '<i class="fa-solid fa-redo"></i> Use Settings'
+    task['useSettings'].style.display = 'block'
```

Comment:
Same question as below - do we really need this line? Is there ever a situation where this button needs to be hidden? If not, we can remove the `display: none` and this `display: block` line, since it'll be visible by default

Reply:
Made the change. Use Settings is now always visible.